### PR TITLE
Include

### PR DESCRIPTION
**Pull request by danielalexlarsen**

### CODE REVIEW

File: src/main/java/org/flowsoft/flowg/nodes/StringNode.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package org.flowsoft.flowg.nodes;
```
Comment (mads256h) [inline src/main/java/org/flowsoft/flowg/nodes/StringNode.java:1]: This file should be removed

File: src/main/java/org/flowsoft/flowg/visitors/IVisitor.java [hunk 2]
```diff
@@ -66,4 +74,6 @@
     T Visit(BooleanLiteralNode booleanLiteralNode) throws TException;
     T Visit(PointNode pointNode) throws TException;
     T Visit(PointEntryNode pointEntryNode) throws TException;
+
+
```
Comment (mads256h) [inline src/main/java/org/flowsoft/flowg/visitors/IVisitor.java:78]: Delete these

**Review by mads256h — CHANGES_REQUESTED** [review]

File: src/main/java/org/flowsoft/flowg/nodes/SysStringNode.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package org.flowsoft.flowg.nodes;
+
+import java_cup.runtime.ComplexSymbolFactory;
+import org.flowsoft.flowg.nodes.base.NullaryNode;
+import org.flowsoft.flowg.visitors.IVisitor;
+
+public class SysStringNode extends NullaryNode<String> {
+
+
```
Comment (Malgreen) [inline src/main/java/org/flowsoft/flowg/nodes/SysStringNode.java:9]: Delete random newline or else

**Review by Malgreen — CHANGES_REQUESTED** [review]

**Review by andreasalstrup — APPROVED** [review]
LGTM

**Review by mads256h — APPROVED** [review]

**Review by Malgreen — APPROVED** [review]

**Pull request merged**